smart_air_conditioning_code_1.ino

```diff
@@ -0,0 +1,48 @@
+#include <ESP8266WiFi.h>
+//#include <ESPAsyncWebServer.h>
+
+#include "IRremote.hpp"
+
+// Replace with your network credentials
+const char *ssid = "wi-fi ssid";
+const char *password = "wi-fi password";
+
+const int irReceiverPin = 5; // D5
+const int irLedPin = 7;  // D7
+
+const int buttonPin = 6; // D6
+
+unsigned long previousMillis = 0UL;

```

Comment:
- [ ] Variable type improvement 
1) Uint32_t
2) no need to double check it's type